Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -60,6 +60,7 @@
 from RecoVertex.BeamSpotProducer.BeamSpot_EventContent_cff import *
 from CommonTools.ParticleFlow.EITopPAG_EventContent_cff import EITopPAGEventContent
 from RecoPPS.Configuration.RecoCTPPS_EventContent_cff import *
+from RecoHGCal.Configuration.TICL_EventContent_cff import *
```

Comment:
please do not fragment the HGCAL event content to more pieces that necessary.
This general file would be best to contain a minimal set per subsystem.
So, ```RecoHGCal.Configuration.RecoHGCal_EventContent_cff``` would be more generic and follow the style of most cases above.

Reply:
This comment still need to be addressed or answered...